doc/classes/AudioEffectEQ.xml

```diff
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectEQ" inherits="AudioEffect" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Base class for audio equalizers. Gives you control over frequencies.
+		Use it to create a custom equalizer if [AudioEffectEQ6], [AudioEffectEQ10] or [AudioEffectEQ21] don't fit your needs.
```

Comment:
I think this may be too much for the brief description...?

Reply:
It is not 200 the max ?

Reply:
I think you can remove "Use it to create a custom equalizer if [AudioEffectEQ6], [AudioEffectEQ10] or [AudioEffectEQ21] don't fit your needs.", it's not that useful.